index.html

```diff
@@ -1,184 +1,288 @@
+        </div>
+      </main>
+      <!------ project 4 --->
+      <main class="project-row">
+        <div class="left">
+          <img src="./assets/project4.svg" alt="project-1" />
+        </div>
+        <div class="right">
+          <h1>Multi-Post Stories</h1>
+          <div class="content">
+            <h3 class="author">FACEBOOK</h3>
+            <img src="./assets/Counter.svg" alt="dot" />
+            <h3>Full Stack Dev</h3>
+            <img src="./assets/Counter.svg" alt="dot" />
+            <h3>2015</h3>
+          </div>
+          <p>
+            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <ul class="tag">
+            <li>HTML</li>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button" class="btn-detail">See project</button>
+        </div>
+      </main>
     </section>
     <section class="about-me">
-        <main class="top">
-            <h1 class="text">About Myself</h1>
-            <p class="intro-text">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-            <div class="socials">
-                <h5>LET'S CONNECT</h5>
-                <ul class="social-icon">
-                    <li> <a href="#"> <img src="./assets/twitter.svg" alt="twitter"> </a> </li>
-                    <li> <a href="#"> <img src="./assets/Linkedin icon.svg" alt="Linkedin"> </a> </li>
-                    <li> <a href="#"> <img src="./assets/medium.svg" alt="medium"> </a> </li>
-                    <li> <a href="#"> <img src="./assets/github.svg" alt="github"> </a> </li>
-                    <li> <a href="#"> <img src="./assets/angellist.svg" alt="angellist"> </a> </li>
-                </ul>
-            </div>
-            <button type="button" class="btn-detail resume">Get my resume</button>
-        </main>
-        
-        <main class="skills">
-            <div class="dropdown">
-                <div class="lang">
-                    <h3>Languages</h3>
-                    <img src="./assets/arrow.png" alt="arrow">
-                </div>
-                <div class="list">
-                    <div class="item">
-                        <img src="./assets/lang1.png" alt="lang1">
-                        <p>JavaScript</p>
-                    </div>
-                    <div class="item">
-                        <img src="./assets/lang2.png" alt="lang2">
-                        <p>HTML</p>
-                    </div>
-                    <div class="item">
-                        <img src="./assets/lang3.png" alt="lang3">
-                        <p>CSS</p>
-                    </div>
-                </div>
+      <main class="top">
```

Comment:
- [OPTIONAL] To follow best semantics and practices, we only use one main tag in entire webpage. that main tag has multiple `section` children in it. kindly consider removing these `main` tags that you are taking inside every section.